README.md

```diff
@@ -1,24 +1,156 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+  <br/>
+  <h3><b>BuddyBudget</b></h3>
 
-Things you may want to cover:
+</div>
 
-* Ruby version
+# 📗 Table of Contents
 
-* System dependencies
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 BuddyBudget ](#-buddybudget-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+    - [Live Link ](#live-link-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Configuration
 
-* Database creation
+# 📖 BuddyBudget <a name="about-project"></a>
 
-* Database initialization
+**BuddyBudget** is a a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+[Click here to watch a video description of the project.](https://www.loom.com/share/760fc775d54640229856ebfa40607faa?sid=2d61c5ea-e91a-40fe-81bf-22cf338f335b)
 
```

Comment:
- [ ] It would be nice if you show some code or feature that you are proud of making to make your readme descriptive and follow the project requirements 👍